beekeeper-metadata-cleanup/pom.xml

```diff
@@ -132,6 +132,7 @@
       <plugin>
         <groupId>com.google.cloud.tools</groupId>
         <artifactId>jib-maven-plugin</artifactId>
+        <version>3.0.0</version>
```

Comment:
The version is configured in egdl parent pom so remove the version from here. This way, all the modules use the same version.